apps/src/aiTutor/interactionsApi.js

```diff
@@ -0,0 +1,26 @@
+export async function savePromptAndResponse(interactionData) {
+  console.log('savePromptAndResponse was called');
+
+  // Checking that the csrf-token exists since it is disabled on test
+  const csrfToken = document.querySelector('meta[name="csrf-token"]')
+    ? document.querySelector('meta[name="csrf-token"]').attributes['content']
+        .value
+    : null;
+
+  console.log('interactionData in interactionsApi', interactionData);
```

Comment:
This is what we have right now: 
![Screenshot 2024-01-29 at 9 44 24 PM](https://github.com/code-dot-org/code-dot-org/assets/12300669/e9fc375b-66d6-4009-a19e-805338d0fdb2)

Which means, this is what we still need: 
- [ ] get the real `userId`
- [ ] get the real `scriptId`
- [ ] figure out why `status` is undefined 
- [ ] decide how we're going to get the `ai_model_version` which is set in the open_ai_chat_helper; maybe just put it somewhere that is shared between the two?
- [ ] find an example where `project_id` and `project_version_id` would not be null and figure out how to pipe the data through to the interactionsApi

Reply:
`user_id` and `ai_model_version` are being set directly in the ai_tutor_interactions_controller now since we can access that information without having to pass it to the frontend and then back again.

Reply:
Updates 🎉 real `scriptId` and `isProjectBacked` fields
![Screenshot 2024-01-30 at 3 41 39 PM](https://github.com/code-dot-org/code-dot-org/assets/12300669/82c83aa7-36d8-455a-aec7-45e70136f059)